<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingDialog</class>
 <widget class="QDialog" name="TrainingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Training</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pronouncePushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ITS.qrc">
       <normaloff>:/Images/Pronunciation.jpg</normaloff>:/Images/Pronunciation.jpg</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="answerOptionsGroupBox">
     <property name="title">
      <string>Answer Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="aRadioButton">
        <property name="text">
         <string>&amp;A)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bRadioButton">
        <property name="text">
         <string>&amp;B)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cRadioButton">
        <property name="text">
         <string>&amp;C)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dRadioButton">
        <property name="text">
         <string>&amp;D)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ITS.qrc"/>
 </resources>
 <connections/>
</ui>
